scripts/cms-any/add-prs-to-project.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+$content = <<<EOT
+name: Add new pull requests to a github project
+
+on:
+  pull_request:
+    types:
+      - opened
+      - ready_for_review
+
+permissions: {}
+
+jobs:
+  add-pr-to-project:
```

Comment:
```suggestion
  addprtoproject:
```
Get it to match gha pr

Reply:
Done